dynamax/linear_gaussian_ssm/models/linear_gaussian_ssm.py

```diff
@@ -78,33 +87,27 @@ def random_initialization(self, key):
 
         return params, param_props
 
-    def initial_distribution(self, params, **covariates):
+    def initial_distribution(self, params, covariates=None):
         return MVN(params["initial"]["mean"], params["initial"]["cov"])
 
-    def transition_distribution(self, params, state, **covariates):
-        inputs = covariates['inputs'] if 'inputs' in covariates else jnp.zeros(self.input_dim)
-        mean = params["dynamics"]["weights"] @ state + params["dynamics"]["input_weights"] @ inputs 
+    def transition_distribution(self, params, state, covariates=None):
+        inputs = covariates if covariates is not None else jnp.zeros(self.input_dim)
```

Comment:
True, we could replace all instances of `inputs` with `covariates` in the lgssm code. I'm a little hesitant to do it now since I don't want to break anything before Kevin's presentation.